Assignment/solution_code/css/style.css

```diff
@@ -0,0 +1,212 @@
+/*
+Project Name: Week 2 Assignment: Layout;
+Client: Relaxr
+Author: Tegan Holcombe
+Developer @GA in NYC
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+     content: ".";
+     display: block;
+     clear: both;
+     visibility: hidden;
+     line-height: 0;
+     height: 0;
+}
+
+.clear {
+  clear: both;
+}
+
+.alignright {
+  float: right;
+  padding: 0 0 10px 10px; /* note the padding around a right floated image */
+}
+
+.alignleft {
+  float: left;
+  padding: 0 10px 10px 0; /* note the padding around a left floated image */
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+body {
+     color: #000;
+     font-size: 12px;
+     line-height: 1.4;
+     font-family: 'Open Sans', sans-serif;
+     background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
+     background-size: cover;
+}
+
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+/* Center the container */
+#container {
+  width: 100%;
+  margin: auto;
+  height: 500px;
+
+}
+
+.header-container {
+  width: 100%;
+  background-color: rgba(3, 48, 72, 0.3);
+}
+
+header {
+  margin: 0 10%;
+  padding-top: 3%;
+  height: 400px;
+}
+
+.tagline {
+  width: 100%;
+  text-align: center;
+  margin-top: 6%;
+}
+
+.quote {
+  background-image: url(../images/testimonial_bg.jpg);
+  height: 220px;
+}
+
+.quote-box {
+  width: 30%;
+  text-align: center;
+  margin: 0 auto;
+}
+
+.Benefits {
+  background-color: #f0efef;
+  height: 220px;
```

Comment:
You've set a fixed height here and a variable margin/padding below (3%) and the text size is fixed as well.  Whilst this works well on a 13" screen, it won't work on bigger screens because the margin and padding will push your content outside of your fixed height.